cli/cmd/install.go

```diff
@@ -419,13 +419,29 @@ func render(w io.Writer, values *l5dcharts.Values, stage string, options valuesp
 // resource is not part of the Helm chart and will not be present when installing
 // with Helm. If stringData is set to true, the secret will be rendered using
 // the StringData field instead of the Data field, making the output more
-// human readable.
-func renderOverrides(values *l5dcharts.Values, namespace string, stringData bool) ([]byte, error) {
+// human readable. If namespace is empty, The same is read from the passed
+// Values
+func renderOverrides(values chartutil.Values, namespace string, stringData bool) ([]byte, error) {
 	defaults, err := l5dcharts.NewValues()
 	if err != nil {
 		return nil, err
 	}
-	values.Configs = l5dcharts.ConfigJSONs{}
+	// Remove unnecessary fields, including fields added by helm's `chartutil.CoalesceValues`
+	values["configs"] = l5dcharts.ConfigJSONs{}
```

Comment:
for consistency with the next two lines, can we just delete this key?  the resulting yaml should be the same, right?

Reply:
Yep, Should be the same